### PR TITLE
Update documents without numbers

### LINKED ISSUE #96: Elements without document number to be fixed
Extraction from #94 

- NBS CIRC e2
- NBS CIRC sup
- NBS CIRC supJun1925-Jun1926
- NBS CIRC supJun1925-Jun1927

### PR DESCRIPTION
**Pull request by mico**
closes #96

### CODE REVIEW

File: update_codes.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+NIST.CSWP.01162020pt: NIST.CSWP.01162020(por)
+NBS FIPS 11-1-Sep30: NBS FIPS 11-1-Sep30/1977
+NBS CRPL c4-4: NBS CRPL 4-4
+NBS CIRC e2: NBS CIRC 2e2
+!ruby/regexp /^NIST FIPS(?! PUB)/: NIST FIPS PUB
+!ruby/regexp /^NBS CIRC sup$/: NBS CIRC 24e7sup
+!ruby/regexp /^NBS CIRC supJun1925-Jun1926$/: NBS CIRC 24e7sup2
+!ruby/regexp /^NBS CIRC supJun1925-Jun1927$/: NBS CIRC 24e7sup3
+!ruby/regexp /NBS[.\s]CS[.\s]e/: NBS CS-E
+!ruby/regexp /(?<=NBS MP )(\d+)\((\d+)\)/: \1e\2
```
Comment (ronaldtse) [inline update_codes.yaml:22]: Should these be put inside a Serie class?

Comment (mico) [reply update_codes.yaml:22]: > Should these be put inside a Serie class?

ok, I'll try

Comment (ronaldtse) [reply update_codes.yaml:22]: Great, e.g. CIRC a class, FIPS a class, etc.

File: lib/nist_pubid/series/nbs_circ.rb [hunk 2]
```diff
@@ -2,6 +2,26 @@ module NistPubid
...
+        super
+      end
+
+      def parse_supplement(code)
+        return "2" if /supJun1925-Jun1926$/.match?(code)
+
+        return "3" if /supJun1925-Jun1927$/.match?(code)
+
+        super
+      end
```
Comment (mico) [inline lib/nist_pubid/series/nbs_circ.rb:24]: Don't feel it as good solution as alternative to 
```
!ruby/regexp /^NBS CIRC sup$/: NBS CIRC 24e7sup
!ruby/regexp /^NBS CIRC supJun1925-Jun1926$/: NBS CIRC 24e7sup2
!ruby/regexp /^NBS CIRC supJun1925-Jun1927$/: NBS CIRC 24e7sup3
```

But cannot get better idea. Maybe you have some? @ronaldtse

Comment (ronaldtse) [reply lib/nist_pubid/series/nbs_circ.rb:24]: This seems to be already resolved, can we merge this? Thanks!

**Pull request closed without merging**